lib/kiev/logger.rb

```diff
@@ -54,8 +56,11 @@ class Logger
         entry.merge!(data)
       elsif !data.nil?
         entry[:message] = data.to_s
+        entry[:status] = ERROR_STATUS if data.to_s.downcase.include?(LOG_ERROR)
       end
 
+      entry[:level] = LOG_ERROR if entry[:status].to_i.between?(400, 599)
```

Comment:
Also, log level is already passed in from the outside as `severity`, why does this need to be set here?

Reply:
> Also, log level is already passed in from the outside as severity, why does this need to be set here?

Log level that is passed from the outside, sets the log level at app level e.g. 

```
module Severity
    # Low-level information, mostly for developers.
    DEBUG = 0
    # Generic (useful) information about system operation.
    INFO = 1
    # A warning.
    WARN = 2
    # A handleable error condition.
    ERROR = 3
    # An unhandleable error that results in a program crash.
    FATAL = 4
    # An unknown message that should always be logged.
    UNKNOWN = 5
  end
```

I need to set the level as in string so it can be filtered in datadog logs.

Reply:
> This formatter should not be coupled to HTTP, as it is also used in background jobs that do not adhere to HTTP's statuses.

For this reason, I custom set the `status` https://github.com/blacklane/kiev/pull/16/files#diff-65ff7abd2edd51bd17339cf140e85b2eR59 so if it is not HTTP, it can still set level error, maybe not optimal, I'm open for suggestions.